lte/gateway/python/integ_tests/s1aptests/test_attach_detach_setsessionrules_tcp_data.py

```diff
@@ -0,0 +1,248 @@
+"""
+Copyright 2020 The Magma Authors.
+
+This source code is licensed under the BSD-style license found in the
+LICENSE file in the root directory of this source tree.
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+"""
+
+import unittest
+import s1ap_types
+import time
+
+from integ_tests.s1aptests import s1ap_wrapper
+from integ_tests.s1aptests.s1ap_utils import SpgwUtil
+from integ_tests.s1aptests.s1ap_utils import SessionManagerUtil
```

Comment:
Can be imported in line 19 itself

Reply:
I raised the PR in draft mode to share with the team so that they can run quick integ tests on the new set interface. This test is quite raw as it still needs to be automated. I will clean up and incorporate your comments.